bot/cogs/search.py

```diff
@@ -130,7 +130,15 @@ async def _basic_search(self, ctx, query: str, category: str) -> None:
 
     @command()
     async def search(self, ctx: Context, category: str, *, query: str) -> None:
-        """Search online for general results."""
+        """Search online for general results.
+           Valid Categories:
+                            - web
+                            - videos
+                            - music
+                            - files
+                            - images
+                            - it 
+                            - maps"""
```

Comment:
Minor formatting adjustments
```suggestion
        """
        Search online for general results.
        Valid Categories:
             - web
             - videos
             - music
             - files
             - images
             - it 
             - maps
        """
```